### PR TITLE
Codestyle changes related to php-cs-fixer update

### PR DESCRIPTION
**Pull request by vladvildanov**

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -355,8 +355,8 @@ public function __get($name)
     }
 
     /**
-     * @param $name
-     * @param $value
+     * @param        $name
+     * @param        $value
```
Comment (tillkruss) [inline src/Client.php:359]: Let's use the type that the php documentation suggests here.

File: src/Client.php [hunk 2]
```diff
@@ -365,7 +365,7 @@ public function __set($name, $value)
     }
 
     /**
-     * @param $name
+     * @param        $name
```
Comment (tillkruss) [inline src/Client.php:368]: Let's use the type that the php documentation suggests here.

File: src/Command/Redis/Container/ContainerInterface.php [hunk 3]
```diff
@@ -18,8 +18,8 @@ interface ContainerInterface
      * Creates Redis container command with subcommand as virtual method name
      * and sends a request to the server.
      *
-     * @param $subcommandID
-     * @param $arguments
+     * @param        $subcommandID
+     * @param        $arguments
```
Comment (tillkruss) [inline src/Command/Redis/Container/ContainerInterface.php:22]: Let's use the type that the php documentation suggests here.

File: src/Connection/RelayMethods.php [hunk 4]
```diff
@@ -32,7 +32,7 @@ public function onFlushed(?callable $callback)
      * @param  string   $pattern
      * @return bool
      */
-    public function onInvalidated(?callable $callback, ?string $pattern = null)
+    public function onInvalidated(?callable $callback, string $pattern = null)
```
Comment (tillkruss) [inline src/Connection/RelayMethods.php:35]: Can you explain this change, or link me to the relevant docs article?

File: tests/PHPUnit/OneOfConstraint.php [hunk 5]
```diff
@@ -44,7 +44,7 @@ protected function matches($other): bool
     }
 
     /**
-     * @param $other
+     * @param         $other
```
Comment (tillkruss) [inline tests/PHPUnit/OneOfConstraint.php:47]: Let's add a type here.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: src/Connection/RelayMethods.php (hunk 4 above)
Comment (vladvildanov) [reply src/Connection/RelayMethods.php:35]: It's related to php-cs-fixer v3.17 release, looks like right now rule set `@Symfony` (the one that we're using) includes new rule, so it doesn't require question mark on type-hint if `null` value is pre-defined.

https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6876

File: src/Client.php [hunk 6]
```diff
@@ -346,7 +346,7 @@ public function createCommand($commandID, $arguments = [])
     }
 
     /**
-     * @param $name
+     * @param                     $name
```
[the remark this answers is not in the document]
Comment (vladvildanov) [reply src/Client.php]: Done

File: src/Client.php (hunk 1 above)
Comment (vladvildanov) [reply src/Client.php]: Done

File: src/Client.php (hunk 2 above)
Comment (vladvildanov) [reply src/Client.php]: Done

File: src/Command/Redis/Container/ContainerInterface.php (hunk 3 above)
Comment (vladvildanov) [reply src/Command/Redis/Container/ContainerInterface.php]: Done

**Review by tillkruss — APPROVED** [review]

**Review by chayim — APPROVED** [review]

**Pull request merged**